moveit_core/robot_state/include/moveit/robot_state/robot_state.h

```diff
@@ -1647,7 +1659,9 @@ class RobotState
    * the body positions will get corrupted.  You need to make a fresh
    * copy of the AttachedBody object for each RobotState you attach it
    * to.*/
-  void attachBody(AttachedBody* attached_body);
+  [[deprecated("Deprecated use the overload requiring an unique_ptr instead")]] void
+  attachBody(AttachedBody* attached_body);
+  // TODO Remove this function
```

Comment:
Deprecated functions are scheduled for removal automatically.
```suggestion
```

Reply:
Removed. Is the automatic removal done by a CI tool?

Reply:
No. Automatic here just meant that deprecated functions are automatically considered by maintainers for removal in some future version. There is no need to explicitly mark it with an extra TODO.